### PR TITLE
feat!: Use the new API project with breaking changes

### PR DESCRIPTION
**Pull request by taorepoara**
## About this PR
Adapt to API breaking changes:

type => _type
action => listener
remove deprecated elements

Wating for https://github.com/lenra-io/ex_component_schema/pull/13 to be merged

## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I did not include breaking changes
- [x] I made my own code-review before requesting one

### I included unit tests that cover my changes
- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### I added/updated the documentation about my changes
- [ ] 📜 README.md
- [ ] 📕 docs/*.md
- [ ] 📓 docs.lenra.io
- [x] 🙅 no documentation needed

### CODE REVIEW

File: libs/application_runner/lib/router.ex [hunk 1]
```diff
@@ -7,28 +7,31 @@ defmodule ApplicationRunner.Router do
         plug(EnsureAuthenticatedAppPipeline)
       end
 
-      scope "/app", ApplicationRunner do
+      scope "/app-api/v1", ApplicationRunner do
```
[the remark this answers is not in the document]
Comment (taorepoara) [reply libs/application_runner/lib/router.ex:10]: It's to make it more understandable. 

The `/app` is to close of the `/apps` used by the server by the webhooks and resources.

File: libs/application_runner/lib/environment/manifest_handler.ex [hunk 2]
```diff
@@ -56,29 +56,27 @@ defmodule ApplicationRunner.Environment.ManifestHandler do
...
-  @default_routes [%{"path" => "/", "view" => %{"type" => "view", "name" => "main"}}]
+  @default_routes [%{"path" => "/", "view" => %{"_type" => "view", "name" => "main"}}]
+
   def handle_call(:get_lenra_routes, _from, state) do
     Logger.debug("#{__MODULE__} handle call for :get_lenra_routes with #{inspect(state)}")
 
     manifest = Map.get(state, :manifest)
 
-    {:reply, get_routes(manifest), state}
+    {:reply, get_exposer_routes(manifest, "lenra"), state}
```
[the remark this answers is not in the document]
Comment (taorepoara) [reply libs/application_runner/lib/environment/manifest_handler.ex:66]: Yes, it's the purpose of this PR ^^

**Review by pichoemr — DISMISSED** [review]

**Pull request merged**